Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2119,7 +2350,7 @@ internal List<AnimationClip> GetAnimationClips()
     }
 #endif
 
-    void OnDestroy() {
+    protected void OnDestroy() {
```

Comment:
why do we need `protected` here ?

Reply:
It was for overriding it on the server, will remove it